paddle/cinn/hlir/framework/op_lowering_util.h

```diff
@@ -31,8 +31,8 @@ ir::Tensor GetTensor(
 
 std::vector<ir::Tensor> CollectInputTensor(
     const Node* node,
-    std::vector<ir::Tensor>& func_args,                       // NOLINT
-    std::unordered_map<std::string, ir::Tensor>& tensor_map,  // NOLINT
+    std::vector<ir::Tensor>* func_args,
```

Comment:
Usually we put input parameters before output parameters.

This problem also happens on some above functions, please try your best to change it.

Reference:
https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs

Reply:
All new or modified parts of this PR have been modified, but some historical codes remain unchanged.